user/src/main/java/com/jumar/user/exceptions/validation/UserValidator.java

```diff
@@ -0,0 +1,18 @@
+package com.jumar.user.exceptions.validation;
+
+import com.jumar.user.dto.CreateUserDto;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+public final class UserValidator {
+    private UserValidator(){}
+
+    public static boolean dtoFieldsAreNull(CreateUserDto createUserDto) {
+        return createUserDto.getFirstName() == null ||
+               createUserDto.getLastName() == null ||
```

Comment:
look into javax validation and NotNull annotations. You will need an @Valid tag in the controller method for it to work I believe.